credentials.py

```diff
@@ -0,0 +1,30 @@
+import os
+from dotenv import load_dotenv
+from dataclasses import dataclass
+from .logs.logger import logger
+
+
+@dataclass
+class Credentials:
+    user_name: str
+    password: str
+    host: int
+    port: int
+    database: str
+
+    def get(self):
+        return {'user_name': self.user_name,
+                'password': self.password,
+                'host': self.host,
+                'port': self.port,
+                'database': self.database}
+
+
+load_dotenv()
+
+cred = Credentials(os.environ.get("USER_NAME"),
```

Comment:
All environment values can be setup as data class properties at the class definition.
e.g.
```python
@dataclass(frozen=True)
class DatabaseConfig:
    """Database configuration"""

    username: str = field(
        default=os.environ.get("DB_USERNAME")
    )
    port: int = field(
        default=os.environ.get("DB_PORT", 5432)
    )
   ...
```